.babelrc

```diff
@@ -1,4 +1,5 @@
 {
-    "presets": ["@babel/preset-react"],
```

Comment:
- [ ] It would be better if there are snapshot tests that are checking if the main components, at least Home, Calculator, and Quote, from Math Magicians mockups are displayed